index.js

```diff
@@ -1,9 +1,9 @@
-module.exports = { 
-    Test: {
-        VlElement: require('./test/e2e/components/vl-element'), 
-        Page: require('./test/e2e/pages/page'), 
-        Config: require('./test/e2e/config'), 
-        Setup: require('./test/e2e/setup'),
-        Cookie: require('./test/e2e/components/cookie')
-    }
-}
\ No newline at end of file
+module.exports = {
+  Test: {
+    VlElement: require('./test/e2e/components/vl-element'),
+    Page: require('./test/e2e/pages/page'),
+    Config: require('./test/e2e/config'),
+    Setup: require('./test/e2e/setup'),
+    Cookie: require('./test/e2e/components/cookie'),
```

Comment:
komma te veel?

Reply:
Nee, is zelfs aan te raden zodat er sneller een volgende kan toegevoegd worden en is volledig valid.

Reply:
dat het valid is wil nog geloven, dat het aan te raden is ...